entity_event/models.py

```diff
@@ -715,6 +717,39 @@ def __str__(self):
         return s.format(entity=entity, source=source, medium=medium)
 
 
+class SubscriptionQuerySet(QuerySet):
+    """A custom QuerySet for Subscriptions.
+    """
+
+    def cache_related(self):
+        """
+        Cache any related objects that we may use
+        :return:
+        """
+        return self.select_related(
+            'medium',
+            'source',
+            'entity',
+            'sub_entity_kind'
+        )
```

Comment:
This easily fits on one line instead of 6, no?

return self.select_related('medium', 'source', 'entity', 'entity_kind')